view/adminhtml/templates/system/config/form/field/onboard_banner.phtml

```diff
@@ -0,0 +1,24 @@
+<?php
+/** @var \Bold\Checkout\Block\System\Config\Form\Field\OnboardBanner $block */
+/** @var Magento\Framework\Escaper $escaper */
+$bannerData = $block->getBannerData();
+?>
+
+<?php if ($bannerData): ?>
+    <div id="bold-checkout-onboard">
+        <div>
+            <h2><?= $escaper->escapeHtml($bannerData->header) ?></h2>
+            <p><?= $escaper->escapeHtml($bannerData->text) ?></p>
+        </div>
+
+        <button
+            type="button"
+            onclick="window.open (
+                '<?= $escaper->escapeUrl($bannerData->link) ?>',
+                'Bold Account Center',
+                'width=1400,height=950')
```

Comment:
would these be better as max width/height?

Reply:
The window.open method doesn't recognize max-width/height but the documentation does say that dimensions will be corrected if necessary so no part of the popup will be positioned offscreen initially.

Reply:
Sounds good.